webui/apps/eidolon-ui2/playwright.config.js

```diff
@@ -10,6 +10,12 @@ module.exports = defineConfig({
         baseURL: 'http://localhost:3000',
         screenshot: 'only-on-failure',
     },
+    projects: [
+        {
+          name: 'chromium',
+          use: { ...devices['Desktop Chrome'] },
+        },
+    ],
```

Comment:
@jahabeebs Which devices do we care about? Declaring this as a way to start thinking about that. The default `projects` in Playwright uses a `chromium` browser, without specifying Chrome or Edge.

Reply:
I suppose Chrome, Safari, and Firefox would be a good start